documentation/src/main/asciidoc/topics/con_xsite_state_transfer.adoc

```diff
@@ -53,6 +53,10 @@ To avoid temporary network disconnects triggering state transfer operations, the
 The status of a backup location must be offline and it must not be included in the cross-site view with JGroups RELAY2.
 ====
 
+The automatic state transfer is not only triggered when the network is broken, but also when a cache starts.
```

Comment:
@infinispan/docs can you take a look, please?

Reply:
few minor edits otherwise lgtm!